README.md

```diff
@@ -17,6 +17,53 @@ Supports:
 - GNU/Linux distributions that include [util-linux](https://github.com/karelzak/util-linux) and [udev](https://wiki.archlinux.org/index.php/udev).
 - Mac OS X.
 
+The `drivelist` core consists of a set of scripts built with technologies that
+are available by default on the target operating systems (like Bash, VBScipt,
+etc). Each of these scripts attempts to get information about the available
+drives and metadata related to them using any medium the target platform
+provides, like a combination of `diskutil`, `/proc/mounts` etc. You can find
+these scripts in the `scripts/` directory.
+
+The scripts then are expected to print to `stdout` all the drive information
+they gathered in a predefined way, based on the YAML language. The type of
+output `drivelist` expects consists of a set of blank line separated blocks,
+each representing a drive by a set of key/value pairs. The exact keys that we
+expect are constantly changing while we keep improving this module, but you can
+see what's the current expected properties by running the platform script that
+applies to your operating system.
+
+This is how the raw output looks in my MacBook Pro at the time of this writing:
+
+```sh
+$ ./scripts/darwin.sh
+device: /dev/disk0
+description: "APPLE SSD SM0256G"
+size: 251000193024
+mountpoints: []
+raw: /dev/rdisk0
+protected: False
+system: True
+
+device: /dev/disk1
+description: "Macintosh HD"
+size: 249779191808
+mountpoints:
+  - path: /
+raw: /dev/rdisk1
+protected: False
+system: True
```

Comment:
The difference is that this one demonstrates the plain output from the platform scripts while the other show the parsed JSON version.